src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper_libuv.cc

```diff
@@ -40,13 +40,17 @@ bool grpc_ares_query_ipv6() {
 bool grpc_ares_maybe_resolve_localhost_manually_locked(
     const char* name, const char* default_port,
     grpc_core::UniquePtr<grpc_core::ServerAddressList>* addrs) {
+#ifdef GRPC_ARES_RESOLVE_LOCALHOST_MANUALLY
```

Comment:
Related to comment below about `grpc_ares_wrapper_windows_inner.cc`, what do you think about moving contents of this file into that one? I think we could then delete the `bool grpc_ares_maybe_resolve_localhost_manually_locked` definitions from `grpc_ares_wrapper_{windows,posix,libuv}` files.

Perhaps `grpc_ares_wrapper_localhost.cc` could have the ifdef logic here, as well as the definition of `inner_maybe_resolve_localhost_manually_locked`?

Reply:
I assume you mean moving this whole function and leaving `grpc_ares_query_ipv6`. In that case would it make sense to just put the function in `grpc_ares_wrapper.cc`?

Reply:
That also sounds fine. I'd tend to put this in `grpc_ares_maybe_resolve_localhost_manually_locked.cc` file though for the very minor reason that `grpc_ares_maybe_resolve_localhost_manually_locked` would be nice to have in the same file as `inner_maybe_resolve_localhost_manually_locked`, and `inner_maybe_resolve_localhost_manually_locked` should probably be static

Reply:
In my change I put them both in `grpc_ares_wrapper` and they're both static because that's where the only call site is.